docs/tutorials/best-practices/speed-up-templates.md

```diff
@@ -0,0 +1,138 @@
+# Speed up your Coder templates and workspaces
+
+October 31, 2024
+
+---
+
+If it takes your workspace a long time to start, find out why and make some
+changes to your Coder templates to help speed things up.
+
+## Monitoring
+
+You can monitor [Coder logs](../../admin/monitoring/logs.md) through the
+system-native tools on your deployment platform, or stream logs to tools like
+Splunk, Datadog, Grafana Loki, and others.
+
+### Coder Observability Chart
+
+Use the [Observability Helm chart](https://github.com/coder/observability) for a
+pre-built set of monitoring integrations.
+
+With the Observability Helm chart, you can monitor [which specific startup
+metrics are a part of the Helm chart?]
```

Comment:
@dannykopping what metrics would you suggest here? Right now my suggestion would be `coderd_provisionerd_job_timings_seconds` and `coderd_provisionerd_jobs_current`.

Reply:
added these - happy to add more post-merge